dom-query-js.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>DOM: Query and Modify</title>
+</head>
+<body>
+
+<h1 id="main-header">Hello World!</h1>
+<ul>
+    <li data-dbid="1">Item 1</li>
+    <li data-dbid="2">Item 2</li>
+    <li data-dbid="3">Item 3</li>
+    <li data-dbid="4">Item 4</li>
+    <li data-dbid="5">Item 5</li>
+</ul>
+<h2 id="sub-header">Hello Again.</h2>
+<p class="sub-paragraph">
+    Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do
+    eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+    ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
+    aliquip ex ea commodo consequat. Duis aute irure dolor in
+    reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+    pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
+    culpa qui officia deserunt mollit anim id est laborum.
+</p>
+
+<script>
+    (function() {
+        "use strict";
+        // TODO: Uncomment next line and get the main header element by id
+         var mainHeader = document.getElementById("main-header");
+
+        // TODO: Set inner html of mainHeader to "JavaScript is Cool"
+
+        mainHeader.innerHTML = "JavaScript is cool";
+        console.log(mainHeader.innerHTML);
+
+        // TODO: Uncomment the next line and get the sub header element by id
+         var subHeader = document.getElementById("sub-header");
+        console.log(subHeader.innerHTML);
+
+        // TODO: Set the text color of subHeader to blue
+        subHeader.style.color = "blue";
+        console.log(subHeader.innerHTML);
+
+        // TODO: Uncomment the next line and get all list items
+         var listItems = document.getElementsByTagName("li");
+         console.log(listItems);
+
+        // TODO: Set text color on every other list item to grey
+        // TODO: Set text color of li with dbid = 1 to blue
+
+        for (var i=0; i < listItems.length; i++){
+            var dbid = listItems [0].style.color = "blue";
+            var skip = listItems [4].style.color = "grey";
+            skip = listItems [2].style.color ="grey";
+        }
+
+        // TODO: Uncomment the next line and get all elements with class name sub-paragraph
+        var subParagraphs = document.getElementsByClassName("sub-paragraph");
+        console.log(subParagraphs);
+
+        // TODO: Change the text in the first sub paragraph to "Mission Accomplished!"
+        subParagraphs[0].innerHTML = "Mission Accomplished";
+        console.log(subParagraphs.innerHTML);
```

Comment:
You are missing a `[0]` somewhere in your `console.log()`. 
When running in a web-browser I get an "undefined" in my javascript console.